libminifi/include/io/BaseStream.h

```diff
@@ -46,13 +46,12 @@ class BaseStream : public DataStream, public Serializable {
       : composable_stream_(this) {
   }
 
-  BaseStream(DataStream *other)
+  explicit BaseStream(DataStream *other)
```

Comment:
This breaks API.
I agree with the change, but if someone depended on BaseStream being implicitly constructible from DataStream, we would break their usage, which we cannot do in a minor release.